website/sidebars.js

```diff
@@ -53,6 +53,7 @@ module.exports = {
         'Plugins': [
             'plugins/api_overview',
             'plugins/ax_sweeper',
+            'plugins/nevergrad_sweeper',
```

Comment:
For now please keep things alphabetical.
I will will probably start grouping by plugin type once i get another plugin or two.